src/Cms/Collection.php

```diff
@@ -94,9 +88,10 @@ public function __unset($id)
 	 * an entire second collection to the
 	 * current collection
 	 *
-	 * @param mixed $object
+	 * @param TValue|self|array $object
```

Comment:
I think instances of child classes would still satisfy `instanceof self` as a child class instance is automatically also an instance of the parent class. So IMO this type hint should be:

```suggestion
	 * @param TValue|static|array $object
```